commit 18aaa82bd36ae3d4eaa3f1d1d8cf643e39f151cd
Author: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
Date:   Thu Aug 22 09:03:20 2024 +0200

    net: netlink: Remove the dump_cb_mutex field from struct netlink_sock
    
    Commit 5fbf57a937f4 ("net: netlink: remove the cb_mutex "injection" from
    netlink core") has removed the usage of the 'dump_cb_mutex' field from the
    struct netlink_sock.
    
    Remove the field itself now. It saves a few bytes in the structure.
    
    Signed-off-by: Christophe JAILLET <christophe.jaillet@wanadoo.fr>
    Reviewed-by: Simon Horman <horms@kernel.org>
    Signed-off-by: David S. Miller <davem@davemloft.net>

diff --git a/net/netlink/af_netlink.h b/net/netlink/af_netlink.h
index 9751e29d4bbb..5b0e4e62ab8b 100644
--- a/net/netlink/af_netlink.h
+++ b/net/netlink/af_netlink.h
@@ -41,7 +41,6 @@ struct netlink_sock {
 	struct netlink_callback	cb;
 	struct mutex		nl_cb_mutex;
 
-	struct mutex		*dump_cb_mutex;
 	void			(*netlink_rcv)(struct sk_buff *skb);
 	int			(*netlink_bind)(struct net *net, int group);
 	void			(*netlink_unbind)(struct net *net, int group);